client/app/nonComp/components/ReportPage/Conditions/IssueType.jsx

```diff
@@ -1,5 +1,52 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { Controller, useFormContext } from 'react-hook-form';
+import { get } from 'lodash';
+import * as yup from 'yup';
+import SearchableDropdown from 'app/components/SearchableDropdown';
+import ISSUE_TYPES from 'constants/ISSUE_TYPES';
 
-export const IssueType = () => {
-  return <div>issue type test component please ignore</div>;
+export const issueTypeSchema = yup.object({
+  issueTypes: yup.string().required('Please select an option')
+});
+
+// Convert to array and sort alphabetically by label
+const formattedIssueTypeCodes = Object.entries(ISSUE_TYPES).map((issueType) => {
+  return {
+    value: issueType[0],
+    label: issueType[1]
+  };
+}).
+  sort((stringA, stringB) => stringA.label.localeCompare(stringB.label));
+
+export const IssueType = ({ control, field, name }) => {
+  const { errors } = useFormContext();
+  const nameIssueTypeCodes = `${name}.options.issueTypeCodes`;
```

Comment:
This name probably needs to match up everywhere with the error codes and schema so it should probably be changed to issueTypes or the errors schema needs to be changed to issueTypeCodes: errorStuff
```suggestion
  const nameIssueTypeCodes = `${name}.options.issueTypes`;
```